.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+jobs:
+  build:
+    docker: # executor type
+      - image: buildpack-deps:trusty
+    # The secondary container is an instance of the second listed image which is run in a common network where ports exposed on the primary container are available on localhost.
+    steps:
+      - add_ssh_keys:
+          fingerprints: 'Your fingerprint'
+      - checkout
+      - run:
+          name: Update npm
+          command: |
+            sudo apt-get install curl
+            curl -sL https://deb.nodesource.com/setup_12.x | bash -
+            sudo apt-get install -y nodejs
+      - run:
+          name: Install Dependence
```

Comment:
Typo in the name

Reply:
This is the name showing in the Circle CI console,I think there is no need to change this?

Reply:
This should be **Install dependencies**  right?

Reply:
Oops, yes I'm a chinese junior school student so I may make mistakes, sorry for that😥

Reply:
Its okay 👍